### DISCUSSION TITLE
want to upgrade the release of existing Winget package.

### DISCUSSION THREAD

**Discussion by a-sajjad72** [microsoft/winget-pkgs#109531] [General]
I want to upgrade the version/release of an existing winget package. i am not the owner/author of that package. how do i contribute to it. package [link](https://github.com/microsoft/winget-pkgs/tree/master/manifests/g/GnuWin32/Grep).

**Comment by mdanish-kh** [comment]
Please follow [Submitting a Package](https://github.com/microsoft/winget-pkgs#submitting-a-package) guide.
